### ISSUE TITLE
ion-rust returns an error for a line comment with no content

### ISSUE THREAD

**Issue by popematt** [amazon-ion/ion-rust#425]
The following Ion cannot be parsed by `ion-rust`:
```
//
{ a: 1 }
```

I tested this using the ion cli. (I'm using `fish`—this might need to be modified for `bash`/`zsh`.)
```fish
echo "//
  { a: 1 }" | ion dump
```
The output I get is:
```
Error: unrecognized input near line 1: '//
'
```
The same result also occurs with
```fish
echo "//" | ion dump
```

If seems that if you add any character other than a newline after the `//`, it parses just fine.

**Comment by zslayton** [comment]
Good find! We really should have a test for this in `ion-tests`.

**Comment by dlurton** [comment]
I have been bitten by this.  I am trying to use `ion_schema` and add comment documentation to my schema.

**Comment by zslayton** [comment]
This was addressed in #474 will go out in the next release.
